pkg/kdump/kdump-view.jsx

```diff
@@ -364,6 +364,12 @@ export class KdumpPage extends React.Component {
                     /* mount targets outside of nfs are too complex for the
                      * current target dialog */
                     kdumpLocation = _("On a mounted device");
+                } else if (target.type == "ftp") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17634-20220902-133021-0d93bc50-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)